src/server/models/item.js

```diff
@@ -334,20 +334,19 @@ schema.methods.setImportDateNow = function() {
 
 schema.methods.normalise = function() {
    var item = this;
-   if (!this.title)
-      this.title = generateTitle(item);
+   if ( ! this.title )
+      this.title = generateTitle( item );
 
    // system-managed fields
-   this.tags = indexHashtags(item);
-   this.lowerCaseTags = _.map(this.tags, tag => tag.toLowerCase());
-   this.textContent = generateTextContent(item);
+   this.tags.push( ...indexHashtags(item) );
```

Comment:
The `push` (append) is causing tags to accumulate each edit, causing duplicate tags. I think this is an unnecessary change; possibly was sending tags through as `tags` not `userTags` in an early iteration.

Reply:
Fixed!